addon/components/weekly-calendar.js

```diff
@@ -0,0 +1,83 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import moment from 'moment';
+
+export default class WeeklyCalendarComponent extends Component {
+  @service intl;
+  @service moment;
+
+  @action
+  scrollView(calendarElement, [earliestHour]) {
+    if (earliestHour === 24 || earliestHour < 2) {
+      return;
+    }
+    calendarElement.scrollTop = this[`hour${earliestHour - 2}`].offsetTop;
```

Comment:
i don't get this - what is happening here with `this`?

Reply:
That deserves a comment for sure, I'll add it.  

All of the hour elements are registering themselves here https://github.com/ilios/common/pull/1177/files?file-filters%5B%5D=.hbs#diff-5d032c49f2dc0c166fff5055177a5573R41. As hour1, hour2, etc. Scrolling to the actual hour didn't work though, for whatever reason I had to scroll back two hours. So this lines gets `offsetTop` from `this.hour7` (7 being two hours before the first session happens).